components/eam/cime_config/config_compsets.xml

```diff
@@ -210,65 +135,22 @@
     <alias>FADIAB</alias>
     <lname>2000_EAM%ADIAB_SLND_SICE_SOCN_SROF_SGLC_SWAV</lname>
   </compset>
-
-  <!-- FG compsets -->
-
-  <compset>
-    <alias>F_ARM97_SCAM</alias>
-    <lname>AR97_EAM%SCAM_ELM%SP_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
-  </compset>
-
-  <compset>
-    <alias>F_SCAM</alias>
-    <lname>SCAM_EAM%SCAM_ELM%SP_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
```

Comment:
F_SCAM is the standard compset for SCM configurations. Need to retain this, but replacing SCAM with SCM and even F_SCAM with FSCM. Ping @bogensch to be aware of this change. Peter, do you have a preference? Should we use F_SCM instead?

Reply:
I already talked to Peter about this and he's ok with this change in the name convention.